content/report_RHF.tex

```diff
@@ -1,9 +1,244 @@
 \section{Introduction}
 \label{sec:intro}
 This report will discuss the notebook we made on restricted Hartree-Fock theory.
- We will walk trough the notebook step by step and discuss what we see there. 
- The same titles will be used, so you can follow along easily. 
- Relevant parts of the code will be displayed.
+We will walk trough the notebook step by step and discuss what we see there. 
+The same titles as in the notebook will be used, as to make sure the report can be 
+easily followed. Relevant parts of the code will be displayed.
+
+ \section{Theoretical Approach}
+ \label{sec:theory}
+ \subsection{Introduction to Theory}
+ \label{subsec:intro}
+ Before we actually dive into the code, we will discuss the theory behind it.
+ The actual goal of this experiment is to calculate the energy of a molecule,
+ mainly to solve the Schrödinger equation for that molecule. This, unfortunately,
+ is impossible. So we have to find an alternative. This alternative would be
+Hartree-Fock theory. Bluntly put we can define another operator, the Fock operator,
+that consists of one-electron operators and two-electron operators. This is 
+displayed in Equation \ref{eq: fock matrix}. 
+
+\begin{equation}\label{eq: fock matrix}
+    \hat{f}(1) = \hat{h}(1) + 2\sum_i^{N/2}\hat{J}_i - \sum_i^{N/2}\hat{K}_i

```

Comment:
Keep in mind that this definition is only valid in the restricted closed shell scheme, but nevertheless correct for this report.